source/SIL.AppBuilder.Portal.Frontend/src/lib/toast.ts

```diff
@@ -14,10 +14,12 @@ export interface NotifyOptions {
 }
 export function show(msg: string, options: NotifyOptions) {
   // notify.show(msg, options.type, options.timeout);
+  console.log(msg, options);
```

Comment:
added these so I could see when the toast messages are shown in the console.

when toasts are actually implemented, we may want to consider using tho context api. not sure yet.